src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -27,6 +27,14 @@ def __str__(self):
         return "\n".join(map(str, self))
 
 
+def validate_max_tokens(max_tokens):
+    # openai only supports max_tokens is larger than 0
+    if max_tokens is None or str(max_tokens).lower() == "inf":
+        return
+    elif int(max_tokens) <= 0:
```

Comment:
A system error would be raised if abc is passed?

Reply:
if tool accepts "asd" as max_tokens, I think user error may be more reasonable

Reply:
checked from portal. 
![image](https://github.com/microsoft/promptflow/assets/75061414/ae7131ba-adf7-4590-87e5-cf7a9abbc472)